### ISSUE TITLE
[BUG] Need to bundle js.map files for django 4+

### ISSUE THREAD

**Issue by cblakkan** [vitalik/django-ninja#533]
**Collectstatic fails on django 4+ if self hosting JS files**

See https://code.djangoproject.com/ticket/33353

Adding `ninja` to `INSTALLED_APPS` causes collectstatic to fail when using whitenoise

```
#0 0.428 Post-processing 'ninja/redoc@2.0.0-rc.66.standalone.js' failed!
#0 0.428
#0 0.428 Traceback (most recent call last):
#0 0.428   File "/opt/app/manage.py", line 22, in <module>
#0 0.428     main()
#0 0.428   File "/opt/app/manage.py", line 18, in main
#0 0.428     execute_from_command_line(sys.argv)
#0 0.428   File "/opt/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
#0 0.428     utility.execute()
#0 0.428   File "/opt/venv/lib/python3.10/site-packages/django/core/management/__init__.py", line 440, in execute
#0 0.428     self.fetch_command(subcommand).run_from_argv(self.argv)
#0 0.428   File "/opt/venv/lib/python3.10/site-packages/django/core/management/base.py", line 402, in run_from_argv
#0 0.428     self.execute(*args, **cmd_options)
#0 0.428   File "/opt/venv/lib/python3.10/site-packages/django/core/management/base.py", line 448, in execute
#0 0.428     output = self.handle(*args, **options)
#0 0.428   File "/opt/venv/lib/python3.10/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 209, in handle
#0 0.429     collected = self.collect()
#0 0.429   File "/opt/venv/lib/python3.10/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 154, in collect
#0 0.429     raise processed
#0 0.429 whitenoise.storage.MissingFileError: The file 'ninja/redoc.standalone.js.map' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0xffff9d419960>.
#0 0.429
#0 0.429 The JS file 'ninja/redoc@2.0.0-rc.66.standalone.js' references a file which could not be found:
#0 0.429   ninja/redoc.standalone.js.map
#0 0.429
#0 0.429 Please check the URL references in this JS file, particularly any
#0 0.429 relative paths which might be pointing to the wrong location.
#0 0.429
```

**Versions (please complete the following information):**
 - Python version: 3.10
 - Django version: 4.1
 - Django-Ninja version: 0.19.1

**Comment by kmasuhr** [comment]
In your django settings change static file storage class to:
`STATICFILES_STORAGE = 'whitenoise.storage.CompressedStaticFilesStorage'`

In django 4.1 there was change: https://docs.djangoproject.com/en/4.1/releases/4.1/#django-contrib-staticfiles

**Comment by cblakkan** [comment]
> In your django settings change static file storage class to: `STATICFILES_STORAGE = 'whitenoise.storage.CompressedStaticFilesStorage'`
> 
> In django 4.1 there was change: https://docs.djangoproject.com/en/4.1/releases/4.1/#django-contrib-staticfiles

This works but completely disables the fingerprinting of assets making them uncacheable... For my current use case that's not a big deal but if a site were to have any frontend with assets it's not a solution...

**Comment by karpiq24** [comment]
Will this issue be resolved any time soon?

**Comment by cblakkan** [comment]
Thanks @chris-mcdo and @vitalik, curious is there an eta for when this might get released?

**Comment by vitalik** [comment]
@cblakkan by the end of the week should land to pypi

**Comment by henningbra** [comment]
Understand the reason behind making ja.map available, but blocking to upgrade to 4.2 as we don't have all the js.map files available. Any chance this could be removed or disable by setting?

**Comment by vitalik** [comment]
> Understand the reason behind making ja.map available, but blocking to upgrade to 4.2 as we don't have all the js.map files available. Any chance this could be removed or disable by setting?

what do you mean by that - which files do you miss ?
